spec/stubs/upload_stubbing.rb

```diff
@@ -0,0 +1,53 @@
+# Our fake application
+class FakeApp
+  def edit_version
+    @ev ||= EditVersion.new
+  end
+end
+
+# Screenshot class shared between ITC Sceenshot and Deliver Screenshot
+class Screenshot
+
+  def initialize(path, language, screen_size = nil)
+    self.path = path
```

Comment:
consider having all arguments here, maybe from a hash instead. That way you would initialize all elements in the same way. Not like https://github.com/lacostej/deliver/pull/1/files#diff-cdd31e535b4ffedc1f0d374e30ba2bbeR59